src/plugins/bubble/_base.scss

```diff
@@ -0,0 +1,426 @@
+	0% {
+		-webkit-transform: scale(0, 1);
+		transform: scale(0, 1); }
+	95% {
+		-webkit-transform: scale(0, 1);
+		transform: scale(0, 1); }
+	100% {
+		-webkit-transform: scale(1, 1);
+		transform: scale(1, 1); } }
+
+@-webkit-keyframes pulseIn {
+	0% {
+		-webkit-transform: scale(1, 1);
+		transform: scale(1, 1); }
+	15% {
+		-webkit-transform: scale(1.15, 1.15);
+		transform: scale(1.15, 1.15); }
+	30% {
+		-webkit-transform: scale(1, 1);
+		transform: scale(1, 1); }
+	65% {
+		-webkit-transform: scale(1.3, 1.3);
+		transform: scale(1.3, 1.3); }
+	100% {
+		-webkit-transform: scale(1, 1);
+		transform: scale(1, 1); } }
+
+@keyframes pulseIn {
+	0% {
+		-webkit-transform: scale(1, 1);
+		transform: scale(1, 1); }
+	15% {
+		-webkit-transform: scale(1.15, 1.15);
+		transform: scale(1.15, 1.15); }
+	30% {
+		-webkit-transform: scale(1, 1);
+		transform: scale(1, 1); }
+	65% {
+		-webkit-transform: scale(1.3, 1.3);
+		transform: scale(1.3, 1.3); }
+	100% {
+		-webkit-transform: scale(1, 1);
+		transform: scale(1, 1); } }
+
+@-webkit-keyframes grow {
+	to {
+		-webkit-transform: translateX(-50%) scale(0);
+		transform: translateX(-50%) scale(0); } }
+
+@keyframes grow {
+	to {
+		-webkit-transform: translateX(-50%) scale(0);
+		transform: translateX(-50%) scale(0); } }
+
+.trans-left {
+	-webkit-animation-delay: 0s;
+	-webkit-animation-duration: 5s;
+	-webkit-animation-iteration-count: 1;
+	-webkit-animation-name: slideInFromRight;
+	-webkit-animation-timing-function: ease-out;
+	-webkit-transform-origin: 100% 50%;
+	animation-delay: 0s;
+	animation-duration: 5s;
+	animation-iteration-count: 1;
+	animation-name: slideInFromRight;
+	animation-timing-function: ease-out;
+	transform-origin: 100% 50%;
+	will-change: scroll-position; }
+
+.trans-pulse {
+	-webkit-animation: .5s linear 3.5s 1 pulseIn, .5s linear 15s 1 pulseIn, .5s linear 30s 1 pulseIn;
+	animation: .5s linear 3.5s 1 pulseIn, .5s linear 15s 1 pulseIn, .5s linear 30s 1 pulseIn;
+	will-change: transform; }
+
+.loader-typing {
+	-webkit-transform: translateX(-50%) translateY(-50%);
+	bottom: 30%;
+	height: 6px;
+	left: 30px;
+	position: absolute;
+	transform: translateX(-50%) translateY(-50%);
+	width: 26px; }
+
+.loader-dot {
+	-webkit-animation: grow .5s ease-in-out infinite alternate;
+	animation: grow .5s ease-in-out infinite alternate;
+	background-color: #444;
+	border-radius: 50%;
+	height: 6px;
+	position: absolute;
+	width: 6px;
+	will-change: transform; }
+
+.loader-dot.dot1 {
+	-webkit-transform-origin: 100% 50%;
+	left: 0;
+	transform-origin: 100% 50%; }
+
+.loader-dot.dot2 {
+	-webkit-animation-delay: .1s;
+	-webkit-transform: scale(.99);
+	animation-delay: .1s;
+	left: 50%;
+	margin-left: -3px;
+	transform: scale(.99); }
+
+.loader-dot.dot3 {
+	-webkit-animation-delay: .2s;
+	animation-delay: .2s;
+	right: 0; }
+
+button.wb-sl {
+	background: #26374a;
+	border: none;
+	cursor: pointer;
+	font-weight: 700;
+	padding: 5px;
+	width: auto !important;
+	z-index: 501;}
```

Comment:
FYI - after this PR is done, we will do it permanently in wet-boew core here: https://github.com/wet-boew/GCWeb/blob/e2ed5bcf9fbfd19df23561676fa2b761fb0dd5b6/src/views/_screen.scss#L11